### PR TITLE
Corrects Certificate chain check filter information

### PR DESCRIPTION
**Pull request by mvenzke-axway**
https://deploy-preview-2187--axway-open-docs.netlify.app/docs/apim_policydev/apigw_polref/cert_validation/#certificate-chain-check-filter

This updates a paragraph in the Certificate chain check filter section which contained incorrect information about what a certificate chain is.  I also expanded the explanation in that paragraph a bit.  A CA does not insert its entire certificate into the certs it signs, only a signature, and you need to actually provide the CA cert(s) to run the cert chain check.

### CODE REVIEW

File: content/en/docs/apim_policydev/apigw_polref/cert_validation.md [hunk 1]
```diff
@@ -471,7 +471,7 @@ It is a trivial task for a user to generate a structurally sound X.509 certifica
...
 
-When a CA issues a certificate, it digitally signs the certificate and inserts a copy of its own certificate into it. This is called a certificate chain. Whenever an application (such as API Gateway) receives a client certificate, it can extract the issuing CA certificate from it, and run a certificate chain check to determine whether it should trust the CA. If it trusts the CA, it also trusts the client certificate.
+When a CA issues a certificate, it digitally signs the certificate and inserts a copy of its own signature into it. Whenever an application (such as API Gateway) receives a client certificate, it can extract the issuing CA certificate's signature from it, and run a certificate chain check to determine whether it was signed by a trusted CA. If it trusts the CA, it also trusts any client certificate with a valid signature from that CA.  This process goes on recursively with each certificate's issuer, until a trusted root CA, which lists the same CN name as both subject and issuer, is encountered.  This list of certificates, from self-signed root CA to leaf is a certificate chain and all valid certificate chains end with a root CA.
```
Comment (cwiechmann) [inline content/en/docs/apim_policydev/apigw_polref/cert_validation.md:474]: I'm not a native speaker, but is **leaf** in this context really correct and makes sense? Perhaps there is a better word we can use.

**Review by cwiechmann — COMMENTED** [review]
Technically @mvenzke-axway is absolutely correct and it was really wrong before.

**Comment by JohnD-Axway** [comment]
This is spot on @andreamussap

File: content/en/docs/apim_policydev/apigw_polref/cert_validation.md (hunk 1 above)
Comment (JohnD-Axway) [reply content/en/docs/apim_policydev/apigw_polref/cert_validation.md:474]: Maybe make it more explicit. "to the leaf certificate"

**Review by JohnD-Axway — APPROVED** [review]

File: content/en/docs/apim_policydev/apigw_polref/cert_validation.md (hunk 1 above)
Comment (mvenzke-axway) [reply content/en/docs/apim_policydev/apigw_polref/cert_validation.md:474]: Leaf is commonly used, but I agree that "leaf certificate" may make things clearer.  This is just an extension of a plant analogy, if you imagine a path through some tree going from root to leaf, but we have enough non-native speaking users that anything that helps make this clearer for people is better.

**Pull request merged**